data/campaigns/Delfadors_Memoirs/scenarios/11_Wasteland.cfg

```diff
@@ -257,7 +257,7 @@
                 [/set_recruit]
                 [modify_side]
                     side=3
-                    share_maps=yes
+                    share_vision=shroud
```

Comment:
I've now done some testing, on both the changed version and on the current 1.16 (without this PR). The two lines are equivalent, neither has any effect.

Chantal's side has shroud=no, which I assume is why `share_vision=shroud` doesn't clear any shroud for Delfador. To share her vision in this event (but not before), her `[side]` needs both of these:
```
        shroud=yes
        share_vision=none
```

My feeling on the LoW parts of this PR is that it would be good to update the deprecated line, but leave a note for @nemaara's rewrite that it doesn't have the intended effect.

Reply:
With the editing, I'm not sure what you're wanting here. Should I be correcting something? Since this is a master-based change, I think it's fine to include corrections (as opposed to just ensuring the behaviour between the old and new syntax matches exactly).

I'll hold off on merging until this is clarified and resolved.

Reply:
Okay, I think I understand what you mean: it appears the intention is to share Chantal's side's vision with the player after the event occurs, but it does not. I verified this for both before and after this PR's changes. I'll correct this with your suggestion.

Reply:
I logged it as #6521, think it's probably worth fixing in 1.16 too.